internal/pkg/tfconf/module.go

```diff
@@ -120,6 +123,26 @@ func CreateModule(path string) (*Module, error) {
 		}
 	}
 
+	//TfOutput ....
+	type TerraformOutput struct {
+		Sensitive bool   `json:"sensitive"`
+		Type      string `json:"type"`
+		Value     string `json:"value"`
+	}
+
+	var terraformOutputs map[string]*TerraformOutput
+	// var settings = *print.Settings
+	fmt.Println(settings.InjectOutputValues)
+	if settings.InjectOutputValues != "" {
+		jsonFile, err := os.Open(settings.InjectOutputValues)
+		if err != nil {
+			fmt.Println(err)
```

Comment:
Errors should be returned as the second return value
```
return nil, fmt.Errorf("caught error while reading the terraform outputs file at %s: %v", settings.InjectOutputValues, err)
```

Reply:
Golang best practices says otherwise homie >>> https://talks.golang.org/2013/bestpractices.slide#4

Reply:
What do you mean?
I don't see a single print in there

Reply:
And it's funny because they have a bug in there :laughing: